README.md

```diff
@@ -1,26 +1,238 @@
 
-This repository contains example notebooks in `R` and `Python` showing how to access datasets available in the Reef 2050 Integrated Monitoring and Reporting Program Data Management System (RIMReP DMS).  
+This repository contains example notebooks in `R` and `Python` showing how to access datasets available in the Reef 2050 Integrated Monitoring and Reporting Program Data Management System (RIMReP DMS).  These notebooks include suggested workflows on how to query datasets to create summary tables, figures, and maps.  
+
+## Table of contents
+- [What is RIMReP DMS?](#what-is-rimrep-dms)
+- [Searching for datasets in RIMReP DMS](#searching-for-datasets-in-rimrep-dms)
+  * [Searching for datasets via STAC](#searching-for-datasets-via-stac)
+  * [Searching for datasets via Pygeoapi](#searching-for-datasets-via-pygeoapi)
+- [Code snippets](#code-snippets)
+  * [Connecting to tabular dataset in S3 bucket](#connecting-to-tabular-dataset-in-s3-bucket)
+  * [Extracting tabular data from S3 bucket](#extracting-tabular-data-from-s3-bucket)
+  * [Extracting gridded data from S3 bucket](#extracting-gridded-data-from-s3-bucket)
+- [Running example notebooks in this repository](#running-example-notebooks-in-this-repository)
+  * [Setting up your machine](#setting-up-your-machine)
+- [Description of example notebooks in repository](#description-of-example-notebooks-in-repository)
+- [Description of scripts in repository](#description-of-scripts-in-repository)
+  
+## What is RIMReP DMS?
+RIMReP DMS is an Open Geospatial Consortium (OGC) API service and analysis-ready, cloud-optimised (ARCO) repository for data and metadata relevant to the management of the Great Barrier Reef. RIMReP DMS offers services to allow the discovery of the data and the interaction with external RIMReP systems.  
+  
+In simple terms, RIMReP DMS is a data portal that aims to be a "one-stop-shop" for all data related to the Great Barrier Reef World Heritage Area, which can be easily accessed by the Great Barrier Reef Marine Park Authority (GBRMPA) to support evidence-based management strategies. All datasets have a standard format regardless of their origin, which not only facilitates access to data, but also their analysis as it removes the need to quality control individual datasets.  
+
+Datasets that have an open licence are publically available in the RIMReP DMS, while datasets that have a restricted licence are only available to users that have been granted access to them by the data provider.  
+  
+## Searching for datasets in RIMReP DMS
+There are two main ways to browse datasets available in the RIMReP DMS:  
+  
+1. Via the SpatioTemporal Asset Catalogs (STAC). STAC provides a common language to describe a range of geospatial information, so that data can be indexed and easily discovered. Our STAC catalogue is available at [https://stac.staging.reefdata.io/](https://stac.staging.reefdata.io/).  

```

Comment:
Might be better if the link is to the STAC Browser? https://stac.staging.reefdata.io/browser/

Reply:
Thanks for that. I have updated this